Dockerfile

```diff
@@ -13,25 +13,23 @@ FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-alpine AS golatest
 
 FROM golatest AS gobase
 COPY --from=xx / /
-RUN apk add --no-cache file git
+RUN apk add --no-cache file git make
 ENV GOFLAGS=-mod=vendor
 ENV CGO_ENABLED=0
 WORKDIR /src
 
 FROM gobase AS buildx-version
 RUN --mount=target=. \
-  PKG=github.com/docker/buildx VERSION=$(git describe --match 'v[0-9]*' --dirty='.m' --always --tags) REVISION=$(git rev-parse HEAD)$(if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi); \
-  echo "-X ${PKG}/version.Version=${VERSION} -X ${PKG}/version.Revision=${REVISION} -X ${PKG}/version.Package=${PKG}" | tee /tmp/.ldflags; \
-  echo -n "${VERSION}" | tee /tmp/.version;
+  echo -n "$(make ldflags)" | tee /tmp/.ldflags ; echo -n "$(make version)" | tee /tmp/.version
 
 FROM gobase AS buildx-build
-ARG LDFLAGS="-w -s"
 ARG TARGETPLATFORM
+RUN xx-go --wrap
 RUN --mount=type=bind,target=. \
   --mount=type=cache,target=/root/.cache \
   --mount=type=cache,target=/go/pkg/mod \
   --mount=type=bind,source=/tmp/.ldflags,target=/tmp/.ldflags,from=buildx-version \
-  set -x; xx-go build -ldflags "$(cat /tmp/.ldflags) ${LDFLAGS}" -o /usr/bin/buildx ./cmd/buildx && \
+  make build CGO_ENABLED=$CGO_ENABLED LDFLAGS="$(cat /tmp/.ldflags)" DESTDIR=/usr/bin && \
```

Comment:
should the target be last there? so

```
make CGO_ENABLED=$CGO_ENABLED LDFLAGS="$(cat /tmp/.ldflags)" DESTDIR=/usr/bin build
```

Also curious; setting `LDFLAGS` here; does `/tmp/.ldflags` contain the `-X` options?